BricksAndHearts.UnitTests/ControllerTests/Admin/AdminControllerTests.cs

```diff
@@ -24,4 +29,33 @@ public void Index_WhenCalledByAnonymousUser_ReturnsViewWithLoginLink()
         result!.ViewData.Model.Should().BeOfType<AdminViewModel>()
             .Which.CurrentUser.Should().BeNull();
     }
+
+    [Fact]
+    public void GetAdminList_ReturnsViewWith_AdminListModel()
+    {
+        // Arrange
+        var fakeAdminService = A.Fake<IAdminService>();

```

Comment:
You could set this up to return fake data e.g. `A.CallTo(() => fakeAdminService.GetAdminLists()).Returns(stuff)` and then check it's correct. Also fine not to, the code is only two lines so don't want to spent ages writing tests for it